src/Symfony/Component/Intl/Countries.php

```diff
@@ -62,6 +100,16 @@ public static function getName(string $country, string $displayLocale = null): s
         return self::readEntry(['Names', $country], $displayLocale);
     }
 
+    /**
+     * Get country name from alpha3 code.
```

Comment:
Gets the country name from its alpha3 code.

Reply:
Changed